spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -60,6 +60,15 @@ class CometExecSuite extends CometTestBase {
     }
   }
 
+  test("offset") {
+    assume(isSpark34Plus, "Dataset.offset is not available before Spark 3.4")
+    withSQLConf(CometConf.COMET_COLUMNAR_SHUFFLE_ENABLED.key -> "true") {
+      checkSparkAnswer(testData.offset(90))
+      checkSparkAnswer(arrayData.toDF().offset(99))
+      checkSparkAnswer(mapData.toDF().offset(99))
+    }
```

Comment:
Hmm, `Dataset.offset` is a 3.4+ API. `isSpark34Plus` cannot make this test buildable with 3.2/3.3. 🤔